Changes.md

```diff
@@ -11,6 +11,9 @@ API
 
 - M33fVectorDataPlug : Added new plug type for specifying arrays of 3x3 matrices.
 - PlugAlgo : `extraDataFromPlug()` now supports M33fPlug and M33fVectorDataPlug.
+- MetadataAlgo :
```

Comment:
I don't have anywhere to hang this, but the commit body is perhaps a little misleading as it says the `persistent` argument is omitted, I think perhaps it means `persistentOnly`?

Reply:
Good spot. As discussed, I've updated this in place : d1709a9.